elements/lisk-bft/src/bft.ts

```diff
@@ -32,6 +34,20 @@ import {
 export const CONSENSUS_STATE_FINALIZED_HEIGHT_KEY = 'bft:finalizedHeight';
 export const EVENT_BFT_BLOCK_FINALIZED = 'EVENT_BFT_BLOCK_FINALIZED';
 
+export const BFTPersistedValuesSchema = {
+	type: 'object',
+	$id: '/BFT',
+	title: 'LiskBFT',
+	properties: {
+		finalizedHeight: {
+			dataType: 'uint32',
+			fieldNumber: 1,
+		},
+	},
+};
+
+codec.addSchema(BFTPersistedValuesSchema);
```

Comment:
This we should put it in the constructor so that we see the error

Reply:
For this one I am not sure if we do it. `codec` Is a module level object and if we want to add/modify anything to it we should do it on module level. What do you think?

Reply:
If the schema is invalid, does it properly throw the error?

Reply:
Not in this branch as we don't have any validation. But in development it will throw error when requiring the file.